draw_graph.py

```diff
@@ -20,12 +20,12 @@ def __init__(self, parent):
         self.axes.set_xlabel("ČAS")
         self.axes.set_ylabel("HODNOTA")
 
-    def draw(self):
+    def draw(self, table):
         x = []
         y = []
-        for i in range(20):
-            x.append(i * 4)
-            y.append(random.randrange(10, 40))
+        for i in table:
+            x.append(i[0])
+            y.append(i[1])

```

Comment:
tu by to malo byt tak ze metoda dostane zoznam values z measurement_data, tam je entica vo formate (time, (value, unit))

Reply:
prerobila som to podla toho este je otazka ci mame value upravovat podla unit, aby mali vstky hodnoty v grafe jednotne jednotky

Reply:
to sme sa este nedohodli, ale v kazdom pripade si to bude riesit measurement_data a tuto sa to robit nebude